File: dataset/output-bk/MISTRAL_codestral-latest/Task73_MISTRAL_codestral-latest.java
// Java
import java.util.Scanner;

public class AdminPassword {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter admin password: ");
        String password = scanner.nextLine();

        if (password.equals("admin123")) {
            System.out.println("Password is valid.");
        } else {
            System.out.println("Password is invalid.");
        }
    }
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task73_MISTRAL_codestral-latest.c
// C
#include <stdio.h>
#include <string.h>

int main() {
    char password[50];
    printf("Enter admin password: ");
    scanf("%s", password);

    if (strcmp(password, "admin123") == 0) {
        printf("Password is valid.
");
    } else {
        printf("Password is invalid.
");
    }

    return 0;
}